framework/base/ArrayableTrait.php

```diff
@@ -11,6 +11,7 @@
 use yii\helpers\ArrayHelper;
 use yii\web\Link;
 use yii\web\Linkable;
+use yii\rest\Serializer;
```

Comment:
Overall that doesn't look too good. Is there another way to solve it?

Reply:
See this https://github.com/yiisoft/yii2/issues/17889#issuecomment-590565449